_source/_includes/blog_post.html

```diff
@@ -5,29 +5,46 @@
     <header class="BlogPost-header">
 
         <time class="BlogPost-date" datetime="{{ post.date | date: '%F'}}">{{ post.date | date: "%B %-d, %Y" }}</time>
-
+        {%- if post.changelog -%}
+        {% assign last_updated_date = post.changelog[0] | split: '=>' | first | date: "%b %-d, %Y" %}
+        <div class="BlogPost-last-updated">
+            Last Updated: <a href="#changelog">{{ last_updated_date }}</a>
+        </div>
+        {% endif %}
         <h1 class="BlogPost-title">
             {% if page.layout == 'blog_index' or page.layout == 'author' %}
             <a href="{{ post.url }}">{{ post.title }}</a>
             {% else %}
             {{ post.title }}
             {% endif %}
         </h1>
-
         <div class="BlogPost-attribution">
             {% if author.avatar %}
             <a href="/blog/authors/{{ post.author }}/">{% img '{{ author.avatar }}' alt:'{{ author.avatar }}' class:BlogPost-avatar %}</a>
             {% endif %}
             <span class="BlogPost-author"><a href="/blog/authors/{{ post.author }}/">{{ author.full_name }}</a></span>
         </div>
-
     </header>
 
     <div class="BlogPost-content">
         {% if page.update-url %}
             {% include update.html title=page.update-title url=page.update-url %}
         {% endif %}
         {{ include.post_content }}
+
+        {%- if post.changelog -%}
+        <p><a id="changelog"></a>
+        <strong>Changelog</strong>:</p>
+        <ul>
+        {% for change in post.changelog %}
+          {%- assign changes = change | split: '=>' -%}
+            <li>{{ changes[0] | date: "%b %-d, %Y" }}:
+                {%- assign change_text = changes[1] | markdownify | remove: '<p>'| remove: '</p>' | remove: '&#8220;' | remove: '&#8221;' | remove: '}' %}
```

Comment:
If I just print `changes[1]`, it has quotes wrapped around it, and a trailing curly brace `}`, that's what there's all the removes.